README.rst

```diff
@@ -20,7 +20,7 @@ Parsel
 
 
 Parsel is a BSD-licensed Python_ library to extract and remove data from HTML_
-and XML_ using XPath_ and CSS_ selectors, optionally combined with
+and XML_ using XPath_ and CSS_ selectors, as well as JSON_ using JMESPath_, optionally combined with
 `regular expressions`_.
```

Comment:
What about:

```suggestion
Parsel is a BSD-licensed Python_ library to extract and remove data from HTML_, 
JSON, and XML_ documents using expressions written in CSS_, JMESPath_, or 
XPath_, optionally combined with `regular expressions`_. 
```

or, to make things more clear while keeping the first paragraph short and to the point:

```suggestion
Parsel is a BSD-licensed Python_ library to extract data from HTML_, JSON, and 
XML_ documents.

It supports:

-   CSS_ and XPath_ expressions for HTML and XML documents

-   JMESPath_ expressions for JSON documents

-   `Regular expressions`_

-   Removing data
```

Reply:
Later I will accept the second approach, but before I wanted to mention that removing data does not seem like a major feature compared to the other 3 mentioned. I would remove it from the first contact the user has with the tool and put it further down.

Reply:
I am fine with that.